.travis.yml

```diff
@@ -221,6 +281,36 @@ matrix:
       env: TOXENV=py3.6-django2.0-mysql-locmem
     - python: 3.6
       env: TOXENV=py3.6-django2.0-mysql-filebased
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-sqlite3-redis
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-sqlite3-memcached
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-sqlite3-pylibmc
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-sqlite3-locmem
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-sqlite3-filebased
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-postgresql-redis
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-postgresql-memcached
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-postgresql-pylibmc
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-postgresql-locmem
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-postgresql-filebased
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-mysql-redis
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-mysql-memcached
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-mysql-pylibmc
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-mysql-locmem
+    - python: 3.6
+      env: TOXENV=py3.6-django2.1-mysql-filebased
```

Comment:
Did you use my small `travis-matrix.py` script at the root of this project? It generates the whole matrix from tox.

Reply:
Sure did, it's brilliant!

Reply:
Great! I was afraid you did it by hand, which would be tiresome and could introduce errors. ^^"